src/scss/_themes.scss

```diff
@@ -19,39 +19,80 @@
 	}
 }
 
-@mixin oTeaserCollectionSpecial() {
+@mixin oTeaserCollectionSpecial($colour) {
 	position: relative;
 	border-top: 0;
 	margin-bottom: 50px;
 
 	&:after {
 		content: '';
 		position: absolute;
-		top: 0px;
+		top: 0;
 		left: 50%;
 		z-index: -1;
 		width: 100vw;
 		height: 100%;
-		background: $_o-teaser-collection-color-special-bg;
+		background: $colour;
```

Comment:
Could this be more specifically applied to `background-color` 

Could save some issues if anything else is added later on as `background` overrides any other properties.